ovh/resource_cloud_project_kube.go

```diff
@@ -148,10 +146,14 @@ func resourceCloudProjectKubeCreate(d *schema.ResourceData, meta interface{}) er
 	params := (&CloudProjectKubeCreateOpts{}).FromResource(d)
 	res := &CloudProjectKubeResponse{}
 
+	if params.UpdatePolicy != nil {
+		return fmt.Errorf("actually this attribute update_policy cannot be set at object creation time, once the object created this attribute can be set and/or updated. This is a temporary bug on our OVH APIV6 side")
```

Comment:
```suggestion
		return fmt.Errorf("The update_policy attribute cannot be set at cluster creation time. Once the cluster is created this attribute can be set and/or updated. This is a temporary bug on our OVH APIV6 side")
```

Reply:
agree but "Error string should not be capitalized or end with punctuation mark" :)